pkg/docker/config/config_test.go

```diff
@@ -88,7 +96,7 @@ func TestGetAuth(t *testing.T) {
 	t.Logf("using temporary home directory: %q", tmpHomeDir)
 
 	configDir1 := filepath.Join(tmpXDGRuntimeDir, "containers")
-	if runtime.GOOS == "windows" || runtime.GOOS == "darwin" {
+	if runtime.GOOS == "windows" || runtime.GOOS == "darwin" || runtime.GOOS == "freebsd" {
```

Comment:
Shouldn’t this be `!= "linux"`, then, to match the code being tested? (In both cases.)